### DISCUSSION TITLE
Welkom bij onze OIV project Discussie!

### DISCUSSION THREAD

**Discussion by OIV-NL-QGIS** [OIV-NL-QGIS/QGIS_project#77] [General]
## We gebruiken Discussies als een plek om in contact te komen met andere leden van onze community. 
We hopen dat u:

Vragen stel die u zich afvraagt.
Deel ideeën.
Ga in gesprek met andere community leden.
Verwelkom anderen en ben ruimdenkend. 
Onthoud dat dit een gemeenschap is die wij zijn
Laten we samen bouwen 💪.

Om aan de slag te gaan, reageer hieronder met een introductie van jezelf en vertel ons wat je doet met deze community.

**Comment by Oostjoost** [comment]
Leuk idee om gebruikers hier discussies te laten voeren en elkaar vragen te stellen

**Reply by OIV-NL-QGIS** [reply]
Ik hoop dat we veel reacties krijgen.
